software/gotenberg.yml

```
@@ -0,0 +1,15 @@
+name: "Gotenberg"
+website_url: "https://gotenberg.dev"
+source_code_url: "https://github.com/gotenberg/gotenberg"
+description: "A Docker-powered stateless API for PDF files."
+licenses:
+  - MIT
+platforms:
+  - Go
+  - Docker
```

Only platforms with installation guides should be listed.

```suggestion
platforms:
  - Docker
```